packages/relay/tests/lib/eth.spec.ts

```diff
@@ -1318,6 +1319,77 @@ describe('Eth calls using MirrorNode', async function () {
       expect(error.message).to.equal('Error encountered estimating the gas price');
     }
   });
+
+  describe('eth_call', async function () {
+    it('eth_call with no gas', async function () {
+      sdkClientStub.submitContractCallQuery.returns({
+            asBytes: function () {
+              return Uint8Array.of(0)
+            }
+          }
+      );
+
+      const result = await ethImpl.call({
```

Comment:
For completeness of coverage this methods should be checking that the result is expected.
Given the return above i think the expected result would be `0x0` or `0x`.

Should make this change for the 4 tests added.

Reply:
Done